Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -364,6 +364,28 @@ public static class Utils {
     }
     #endregion
 
+    #region Array Utils
+
+    /// <summary>
+    /// Prints the elements of an array in a bracket-enclosed, comma-delimited list,
+    /// prefixed by the elements' type.
+    /// </summary>
+    public static string ToArrayString<T>(this T[] arr) {
```

Comment:
How about using `IList<T>` instead of `T[]`, that way you catch arrays _and_ List<T> objects?  Or hell, how about IEnumerable<T> to catch everybody since this is going to allocate a bunch of stuff anyway?

Reply:
yeah that's a good idea. This allocates garbage like nobody's business anyway. I'll change it to IEnumerable